Assignment1/q1.rb

```diff
@@ -0,0 +1,22 @@
+#1. WAP to display number is prime or not
+
+puts "Enter number"
+
+number = gets.to_i
+flag = 0
+if number > 1
+  for i in 2 ... number do
+
+   if number % i == 0
+    flag = 1
+
```

Comment:
Once you get the divisible number, you can break the loop; there is no need to check further as the number is not prime.